<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculate</class>
 <widget class="QMainWindow" name="Calculate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="y_tinput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_tinput_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_tinput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_tinput_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_tinput_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Angle</string>
    </property>
   </widget>
   <widget class="QLabel" name="target3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="target1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_tinput_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_tinput_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="target2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_tinput_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="zPos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_tinput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listRotation">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>341</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="set_target">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Targets</string>
    </property>
   </widget>
   <widget class="MyGLWidget" name="myGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>951</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="yRotSlider">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>590</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="zRotSlider">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>620</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="xRotSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="zPos_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>620</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPos_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>560</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPos_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>590</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addtarget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add target</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculatePath">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Path</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>draw()</slot>
    <slot>addTarget()</slot>
    <slot>paintGL()</slot>
    <slot>addPath()</slot>
    <slot>addPath_bt_clicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>xRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addtarget</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>paintGL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculate</sender>
   <signal>clicked()</signal>
   <receiver>listRotation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CalculatePath</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>addPath_bt_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>687</y>
    </hint>
    <hint type="destinationlabel">
     <x>1014</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
